tasks/hydra-collections-dev.rake

```diff
@@ -11,6 +18,8 @@ end
 
 desc 'Start solr, fcrepo, and run specs'
 task ci: ['engine_cart:generate'] do
```

Comment:
I wonder if we should add the rubocop task here instead to fail quickly (before EC does its thing) if there are style issues.